### DISCUSSION TITLE
Welcome to dw-link Discussions!

### DISCUSSION THREAD

**Discussion by felias-fogg** [felias-fogg/dw-link#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ccrause** [comment]
Congratulations! This looks like good quality work and excellent documentation.  You have some brilliant new ideas which I would like to copy into my own debugwire implementation.

**Comment by felias-fogg** [comment]
Thanks for the flowers! Sure, go ahead and copy things. That is what makes Open Source so much fun. I actually copied probably more than 50% from other implementations.
